### ISSUE TITLE
Ansible playbook for cache db setup

### ISSUE THREAD

**Issue by gpfrancis** [lsst-uk/ARCHIVE-lasair-lsst#26]
Create a playbook for setting up the Sherlock wrapper cache database (SQL snippet currently in tests).

**Comment by tms-epcc** [comment]
14/JUL/21
- @gpfrancis explained this is currently blocked by a Sherlock issue

**Comment by tms-epcc** [comment]
06/OCT/21
- as explained by @gpfrancis via email, this is not a priority for Lasair cycle-5 and hence is moved into the backlog.

**Comment by gpfrancis** [comment]
Cache is not used in present setup due to other issues so this is not currently needed.
